packages/bdd-tests/features/createprescription.feature

```diff
@@ -3,17 +3,17 @@ Feature: Creating a prescription
   Background:
     Given I am authenticated
 
-  @AEA-3116
+  @excluded @AEA-3116
   Scenario Outline: Create 1 line item prescription
     When I create 1 prescription(s) for FGC1 with details
       | snomedId   | medItem | quantity | dosageInstructions        |
       | <snomedId> | <medItem>      | <quantity>       | <dosageInstructions> |
     Then I get a success response 200
 
     Examples:
-      | snomedId          | medItem                       | quantity | dosageInstructions        |
-      | 322341003         | High-strength Co-codamol 30mg | 20       | 2 times a day for 10 days |
-      | 39732311000001104 | Amoxicillin 250mg Capsules    | 22.5     | 1 time a day for 11 days  |
+      | snomedId  | medItem                       | quantity | dosageInstructions        |
+      | 322341003 | High-strength Co-codamol 30mg | 20       | 2 times a day for 10 days |
+      #| 39732311000001104 | Amoxicillin 250mg Capsules    | 22.5     | 1 time a day for 11 days  |
```

Comment:
There are valid tests, not switched on at the moment because I don't want to be running all examples, once the tests are running. There extra test will be run. if you don't want to run a scenario, then they need to be commented out - which is what I am doing here